plugin/restore/mysql/mysql.go

```diff
@@ -107,38 +114,45 @@ func (r *Restore) SwapPITRDatabase(ctx context.Context, database string, timesta
 	renameStmt := fmt.Sprintf("RENAME TABLE %s;", strings.Join(tableRenames, ", "))
 
 	if _, err := txn.ExecContext(ctx, renameStmt); err != nil {
-		return err
+		return "", "", err
 	}
 
 	if err := txn.Commit(); err != nil {
-		return err
+		return "", "", err
 	}
 
-	return nil
+	return pitrDatabaseName, pitrOldDatabase, nil
 }
 
-// DeleteOldDatabase deletes the old database after the PITR swap task.
-func (r *Restore) DeleteOldDatabase(ctx context.Context, database string, suffixTs int64) error {
+// DeletePITRDatabases deletes the temporary pitr databases after the PITR swap task.
+func (r *Restore) DeletePITRDatabases(ctx context.Context, database string, suffixTs int64) error {
 	db, err := r.driver.GetDbConnection(ctx, "")
 	if err != nil {
 		return err
 	}
 
-	pitrOldDatabase := getPITROldDatabaseName(database, suffixTs)
-	if _, err := db.ExecContext(ctx, fmt.Sprintf("DROP DATABASE `%s`;", pitrOldDatabase)); err != nil {
+	pitrOldDatabaseName := getPITROldDatabaseName(database, suffixTs)
```

Comment:
I am too chicken to drop the original database here.

Instead, we should indicate from the UI this is the old database and let users take care of it in their best interest.

Reply:
Then it renders the third "delete" PITR task useless. WDYT? @d-bytebase

Reply:
Only drop the ephemeral PITR database here.
Dropping the original database is done in the 3rd task. Will discuss further.